CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
```

Comment:
This looks good. The only thing for future cases is to be careful that signature changes are breaking e.g.
```python
def method(paramOne, paramTwo):
```
to
```python
def method(param_one, param_two)
```

breaks code for someone who used kw args e.g.
```python
method(paramTwo=value,paramOne=value)
```
or those who had to do that because of partial method definition

```python
partial(method, paramOne=value)
```

For this reason, state in the Changelog that this a breaking change

Reply:
Thanks Stephen for the comments. Updated doc strings, removed commented codes and added breaking changes